cmake/GrpcTargets.cmake

```diff
@@ -126,5 +126,11 @@ function(add_grpc_library NAME)
   add_library(${NAME} STATIC ${generated_sources})
   target_compile_options(${NAME} PUBLIC -Wno-unused-parameter)
   target_include_directories(${NAME} SYSTEM PUBLIC ${include_paths})
-  target_link_libraries(${NAME} PUBLIC userver-grpc Protobuf)
+  find_package(gRPC REQUIRED)
+  if(gRPC_VERSION VERSION_GREATER_EQUAL "1.41")
```

Comment:
This is the earliest version I can see that has the explicit grpc/libabsl dependency (it looks like this comes into the Debian tree at version 1.44) but from CMakeLists.txt it might have been introduce some time ago (well before 1.41)